router/schema_definitions.json

```diff
@@ -0,0 +1,112 @@
+{
+  "routes": {
```

Comment:
Curious, is there a plan on how these def files will be synced with the js implementation?